AutomatedTesting/Gem/PythonTests/Prefab/tests/PrefabTestUtils.py

```diff
@@ -251,16 +251,17 @@ def validate_spawned_entity_transform(entity, expected_position, expected_rotati
     spawned_entity_transform = entity.get_world_translation()
     spawned_entity_rotation = entity.get_world_rotation()
 
-    x_rotation_success = math.isclose(spawned_entity_rotation.x, expected_rotation.x,
-                                      rel_tol=1e-5)
-    y_rotation_success = math.isclose(spawned_entity_rotation.y, expected_rotation.y,
-                                      rel_tol=1e-5)
-    z_rotation_success = math.isclose(spawned_entity_rotation.z, expected_rotation.z,
-                                      rel_tol=1e-5)
+    position_success = wait_for_condition(lambda: spawned_entity_transform == expected_position, 3.0)
```

Comment:
should position also be compared using math.isclose() to avoid precision errors?

Reply:
I haven't hit any issues with position. I think the precision issues with rotation are due to the radians conversion.